frontend/src/metabase/visualizations/lib/utils.js

```diff
@@ -121,13 +121,26 @@ export function computeSplit(extents, left = [], right = []) {
   }
 }
 
-const FRIENDLY_NAME_MAP = {
+const AGGREGATION_NAME_MAP = {
   avg: t`Average`,
   count: t`Count`,
   sum: t`Sum`,
   distinct: t`Distinct`,
   stddev: t`Standard Deviation`,
 };
+const AGGREGATION_NAME_REGEX = new RegExp(
+  `^(${Object.keys(AGGREGATION_NAME_MAP).join("|")})(_\\d+)?$`,
+);
+
+export function getFriendlyName(column) {
```

Comment:
Are some of the changes from https://github.com/metabase/metabase/pull/9429/files#diff-24925d6209e59d8b6faeafcf4c035405R135 in here as well?

Reply:
Ah, yeah I merged that branch in. I might revert that.